lib/parser.js

```diff
@@ -187,6 +209,20 @@ function scope (scanner) {
   }
 }
 
+/*
+ * <scopeEnd>    ::= ")"
```

Comment:
Just to match the rest of the grammar, I'd probably make it `<scope-start>`/`<scope-end>`.